contributor-docs/adrs/adr-004-children-as-api.md

```diff
@@ -2,7 +2,10 @@
 
 ## Status
 
-Approved 2022-05-10
+| Stage    | Status |
+| -------- | ------ |
+| Approved | ✅     |
+| Adopted  | ❌     |
```

Comment:
If we have an issue for adoption, might it be useful to include a link in the `Adopted` status column? And if we don't have an issue, we probably should!

Reply:
Strong agree!